packages/frontend/src/creation-form/components/rewards/index.tsx

```diff
@@ -124,6 +110,14 @@ export const Rewards = ({
         setRewardTokenPickerOpen(false);
     }, []);
 
+    const handleRewardPick = useCallback(
+        (reward: TokenInfoWithBalance): void => {
+            setRewardToken(reward);
+            setRewardMinimumPayout(0n);
```

Comment:
I really believe we shouldn't force the input value to 0 and instead convert undefined to 0 in the `onNext` step if the minimum payout is indeed not specified. Wdyt?